admin/services/check-form.service.js

```diff
@@ -332,6 +332,23 @@ const checkFormService = {
 
   removeWindowAssignment: async (formId, windowId) => {
     return checkFormDataService.sqlRemoveWindowAssignment(formId, windowId)
+  },
+
+  getParsedCheckForms: async (batchIds) => {
```

Comment:
I suggest we just call this `getCheckFormsByIds` or `getCheckForms`.  At any rate, I don't see the need to mention parsing it in the name - the return value from a service should almost certainly be something usable from this tier.

Reply:
@jon-shipley changed to getCheckFormsByIds as it is more descriptive to the method.